lib/src/Base/Func/EnumerateFunctionImplementation.cxx

```diff
@@ -148,5 +148,10 @@ void EnumerateFunctionImplementation::load(Advocate & adv)
     upperBound_ = Indices(getDimension(), std::numeric_limits<UnsignedInteger>::max());
 }
 
+/* Returns the marginal enumerate function */
+EnumerateFunctionImplementation EnumerateFunctionImplementation::getMarginal(const Indices &) const
```

Comment:
No should 
```cxx
EnumerateFunction EnumerateFunctionImplementation::getMarginal(const Indices &) const
```

Reply:
Thank you very much for the detailed suggestions. Fixed at 8ba0bdcdc775a35dbdf8f2987f7701ee352fa488.